README.md

```diff
@@ -16,3 +16,404 @@ const contentDb = new CohContentDb();
+            (the-once-and-future-king.png)
+            
+        3.  Use the Hero version of the badge name, even if the badge is earned as a Villain.  
+            (grounded.png, NOT pilot.png)
+            
+        4.  Append with -h and -v if there are different images for the Hero and Villain versions.  
+            (tested-the-water-h.png and tested-the-water-v.png)
+            
+        5.  Remove special characters (the-doctors-ally.png)
+            
+        6.  Note: Images that are used by multiple badges may have a more generic name. Review existing badges and follow naming conventions as best as possible if you need to add a new image that’s used by multiple new badges. For example, there were four exploration badge imagess: hazard.png, hero.png, villain.png, and praetorian.png. Adding the Labyrinth of Fog exploration badge, one might use labyrinth.png.
+            
+7.  Upload the new badge image files to your working Branch on GitHub.  
+    docs/images/badges/(appropriate category directory, e.g. accolade, exploration, etc.)
+    
+### Updating Category Files
+ATC can provide sequential order. _Only needed if adding a new badge. Skip this section if you’re merely updating an existing badge._
+
+1.  Open the file that starts with an underscore that is in the appropriate category directory. For example: if you’re adding a new Accolade, open the src/badge/accolade/\_accolade-badges.ts file. This step is necessary to get the badge to appear on the Badger site.
+    
+2.  Add an import command for the new badge to the bottom of the first list. This list uses “Export Name” and “Key Name” in the same line. “Export Name” is the Hero Male variant, CamelCase (ProperCase), no spaces or special characters.  
+    Example: _**import {HistoryInTheMaking} from "./history-in-the-making";**_
+    
+3.  Insert the Export Name into the second list in this file, in the same spot the badge appears in-game. Open the appropriate badge category in the in-game list (NOT “Most Recent” – go to the actual category) and see what other badges the new badge falls between. Add the export name for the new badge in the appropriate place in the list.
+    
+### Adding New Badge .ts File
+
+_Only add new .ts file if adding a whole new badge. If updating existing badge, find the appropriate .ts file and edit the required fields. For new badges, you can create a new .ts file from scratch, but it’s strongly recommended that you copy one from the most-similar badge instead._
+
+Each indented (block-quoted) line below is an example of what you should include or modify in badge .ts files (each on separate lines). A full example file without notes is included at the end. The individual sections below use different badges as examples so don’t expect consistency until the mock .ts file at the end.
+&nbsp;<BR>
+&nbsp;<BR>
+> import {ALIGNMENT\_ANY, Alternate, BadgePartialType, BadgeType, IBadgeData} from "coh-content-db";
```

Comment:
Rather than quote blocks, use code blocks for the parts that contain code.

wrap in a single backtick (`) for inline and three backticks (```) for code blocks, like the following:

````
```
import {ALIGNMENT\_ANY, Alternate, BadgePartialType, BadgeType, IBadgeData} from "coh-content-db";
```
````

You also shouldn't need to worry about the BRs or non-breaking spaces then 👍🏻